packages/platform-android/src/commands/runAndroid/tryLaunchEmulator.ts

```diff
@@ -0,0 +1,80 @@
+import execa from 'execa';
+import Adb from './adb';
+
+const getEmulators = () => {
+  try {
+    const emulatorsOutput = execa.sync('emulator', ['-list-avds']).stdout;
+    return emulatorsOutput.split('\n').filter(name => name !== '');
+  } catch {
+    return [];
+  }
+};
+
+const launchEmulator = async (emulatorName: string, adbPath: string) => {
+  return new Promise((resolve, reject) => {
+    const cp = execa('emulator', [`@${emulatorName}`], {
```

Comment:
`emulator` command may not be available if someone doesn't have `$ANDROID_HOME` added to shell path, which is not mandatory (the variable needs to exists tho). 
```suggestion
    const cp = execa(
      process.env.ANDROID_HOME 
        ? `${process.env.ANDROID_HOME}/emulator/emulator` 
        : 'emulator', 
      [`@${emulatorName}`],
    {
```
Would be nice to also catch if both branches failed for some reason, and give a message like: 
> Couldn't launch Android Emulator. Please make sure $ANDROID_HOME env variable is set or the command "emulator" exists.

Reply:
How we can check if it failed because of a missing path? check exit code or check if binary exists before even launching?

Reply:
I think the check I proposed should be just enough